lib/configuration/configuration.js

```diff
@@ -112,10 +112,12 @@ class Configuration {
   static async fetchConfigFile (context) {
     let github = context.github
     let repo = context.repo()
+    let globalSettings = context.globalSettings
+
     // using pull request configuration by default, this behaviour
     // can be controlled using USE_CONFIG_FROM_PULL_REQUEST environment variable
     let usePullRequestConfig = true
-    if (process.env.USE_CONFIG_FROM_PULL_REQUEST !== undefined && process.env.USE_CONFIG_FROM_PULL_REQUEST === 'false') {
+    if (globalSettings.use_config_from_pull_request !== undefined && globalSettings.use_config_from_pull_request === 'false') {
```

Comment:
```suggestion
    if (globalSettings.useConfigFromPullRequest !== undefined && globalSettings.useConfigFromPullRequest === 'false') {
```
it should be camel case here as well no? and also [lib/configuration/configuration.js:163](https://github.com/hellofresh/mergeable/pull/38/files#diff-ece2cbf16d3e26b7f1d2c81ce64c07293b4ea04128f1fa890bae77f00aeda508R163) and [lib/configuration/configuration.js:173](https://github.com/hellofresh/mergeable/pull/38/files#diff-ece2cbf16d3e26b7f1d2c81ce64c07293b4ea04128f1fa890bae77f00aeda508R173)

Reply:
I've typed in the other question too, but these are coming from yaml (so it's just yaml parser variables which is assigned to context) and not ideal to make yaml in camelCase, otherwise we need to change everything . All yaml is `underscored`

Reply:
yes, I know, but we could map it

Reply:
another thing, as it's coming from yaml, it does not need to be a string boolean anymore, wdyt?

Reply:
Yeah for the tests, I've misunderstood something but now I've fixed. To transform the objects, I don't see a reason. Because configuration also has `_` spelling but not camel case (like `must_include`)